### ISSUE TITLE
SCORM and/or xAPI support?

### ISSUE THREAD

**Issue by flipace** [nekomy/nekomy-platform#35]
hi, first off - this looks really interesting, and I'm inclined to look deeper into this project.

before i get on with it however, i wanted to ask whether nekomy does or will support scorm in any way? (uploading scorm packages, providing the scorm api for the courses then)

thanks in advance!

**Comment by gazpachu** [comment]
Yes, it could support scorm, if we find more devs to help in the project.
